### PR TITLE
feat: update squad image placeholder

### PR DESCRIPTION
**Pull request by capJavert**
Frontend PR https://github.com/dailydotdev/apps/pull/2151

### CODE REVIEW

File: src/migration/1672583245776-SourceImageDefault.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export class SourceImageDefault1672583245776 implements MigrationInterface {
     await queryRunner.query(`ALTER TABLE "source"
       ALTER COLUMN "image" SET NOT NULL`);
     await queryRunner.query(`ALTER TABLE "source"
-      ALTER COLUMN "image" SET DEFAULT 'https://daily-now-res.cloudinary.com/image/upload/v1672041320/squads/squad_placeholder.jpg'`);
+      ALTER COLUMN "image" SET DEFAULT 'https://daily-now-res.cloudinary.com/image/upload/s--LrHsyt2T--/f_auto/v1692632054/squad_placeholder_sfwkmj'`);
```
Comment (omBratteng) [inline src/migration/1672583245776-SourceImageDefault.ts:10]: Shouldn't it be a new migration, instead of altering an already existing migration?

Comment (rebelchris) [reply src/migration/1672583245776-SourceImageDefault.ts:10]: Yeah it should be a new migration, this won't trigger

**Review by rebelchris — CHANGES_REQUESTED** [review]
We need to add a new migration or manually set it in DB

File: src/migration/1672583245776-SourceImageDefault.ts (hunk 1 above)
Comment (capJavert) [reply src/migration/1672583245776-SourceImageDefault.ts:10]: I know it wont trigger, I just thought there is no point in updating defaults with current squads. 

I'll add the new one if needed 👍

Comment (rebelchris) [reply src/migration/1672583245776-SourceImageDefault.ts:10]: Ah yes, but we want the new default to be something correct?
So if squads get created without image it needs to default to this new one.

Comment (capJavert) [reply src/migration/1672583245776-SourceImageDefault.ts:10]: Yes, mine oversight, will take care od it.

**Review by rebelchris — APPROVED** [review]
Good to go

**Review by omBratteng — APPROVED** [review]

**Review by idoshamun — APPROVED** [review]

**Pull request merged**